services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].module.scss

```
@@ -43,3 +44,28 @@
     width: 50%;
   }
 }
+
+.tabs-column {
+  position: sticky;
+  z-index: 9000;
```

you can import like: `@use '@carbon/react/scss/utilities/z-index' as z-index;`
```suggestion
  z-index: z-index.z('overlay');;
```